runtime-config.md

```diff
@@ -1,6 +1,12 @@
 ## Mount Configuration
 
-Additional filesystems can be declared as "mounts", specified in the *mounts* array. The parameters are similar to the ones in Linux mount system call. [http://linux.die.net/man/2/mount](http://linux.die.net/man/2/mount)
+Additional filesystems can be declared as "mounts", specified in the *mounts* object.
+Keys in this object are names of mount points from portable config.
+Values are objects with configuration of mount points.
+The parameters are similar to the ones in Linux mount system call.
+[http://linux.die.net/man/2/mount](http://linux.die.net/man/2/mount)
```

Comment:
Instead of splatting in the URL, I'd use: “`the ones in [the Linux mount system call](http://man7.org/linux/man-pages/man2/mount.2.html)`”.  That's also using Kerrisk's site for the page source, since it seems the most canonical.